packages/airnode-node/src/api/index.ts

```diff
@@ -300,28 +301,33 @@ export async function callApi(payload: ApiCallPayload): Promise<LogsData<ApiCall
   const {
     aggregatedApiCall: { parameters },
   } = payload;
-  const ois = payload.config.ois.find((o) => o.title === payload.aggregatedApiCall.oisTitle)!;
+  const ois = payload.config.ois.find((o) => o.title === payload.aggregatedApiCall.oisTitle)! as OIS;
   const endpoint = ois.endpoints.find((e) => e.name === payload.aggregatedApiCall.endpointName)!;
-  const processedParameters = await preProcessApiCallParameters(endpoint, parameters, {
+  const { apiCallParameters: processedApiCallParameters } = await preProcessApiCallParameters(endpoint, parameters, {
     totalTimeoutMs: PROCESSING_TIMEOUT,
   });
 
   // skip API call if operation is undefined and fixedOperationParameters is empty array
   if (!endpoint.operation && isEmpty(endpoint.fixedOperationParameters)) {
-    // contents of preProcessingSpecifications or postProcessingSpecifications (or both) will simulate an API when API call is skipped
-    if (isEmpty(endpoint.preProcessingSpecifications) && isEmpty(endpoint.postProcessingSpecifications)) {
-      const message = `Failed to skip API call. Ensure at least one of 'preProcessingSpecifications' or 'postProcessingSpecifications' is defined and is not an empty array at ois '${payload.aggregatedApiCall.oisTitle}', endpoint '${payload.aggregatedApiCall.endpointName}'.`;
+    // Some processing needs to be defined.
```

Comment:
https://github.com/api3dao/airnode/issues/1915 but I didn't want to do it now.

Reply:
As @dcroote pointed out, the check can be removed because it's verified in the OIS.